lib/resty/core/regex.lua

```diff
@@ -154,6 +156,24 @@ local function get_max_regex_cache_size()
 end
 
 
+function ngx.re.opt(option, value)
+    if option == "jit_stack_size" then
+        local result = C.ngx_http_lua_set_jit_stack_size(value)
+
+        if result == -1 then
```

Comment:
I believe we also have named constants like `FFI_ERROR` and etc. Please avoid magical numbers here as well. Thanks!